docs/2.guide/2.directory-structure/1.server.md

```diff
@@ -494,3 +494,22 @@ export default defineNuxtConfig({
 })
 ```
 ::
+
+
+
+
+### Cavets on Auto import/export
+
+Not every directory in `./server` is imported into other files.
+
+`Wizard` is exported from this file automatically.
+
+```ts [./server/apis/wizards.get.ts]
```

Comment:
This is the same name as the one bellow, did you meant utils ?

Reply:
My bad, I meant to specifically point out that Models & models are different/case sensitive from my observation so far.